File: php/src/Cnpj.php
<?php

declare(strict_types=1);

namespace LibCnpj;

final class Cnpj
{
    public static function isValid(string $value): bool
    {
        return CnpjValidator::isValid($value);
    }

    public static function format(string $value): string
    {
        return CnpjFormatter::format($value);
    }

    public static function strip(string $value): string
    {
        return CnpjFormatter::strip($value);
    }

    public static function isFormatted(string $value): bool
    {
        return CnpjFormatter::isFormatted($value);
    }

    public static function calculateCheckDigits(string $base): ?string
    {
        return CnpjValidator::calculateCheckDigits($base);
    }
}

File: java/src/main/java/com/libcnpj/Cnpj.java
package com.libcnpj;

public final class Cnpj {

    private Cnpj() {
    }

    public static boolean isValid(String value) {
        return CnpjValidator.isValid(value);
    }

    public static String format(String value) {
        return CnpjFormatter.format(value);
    }

    public static String strip(String value) {
        return CnpjFormatter.strip(value);
    }

    public static boolean isFormatted(String value) {
        return CnpjFormatter.isFormatted(value);
    }

    public static String calculateCheckDigits(String base) {
        return CnpjValidator.calculateCheckDigits(base);
    }
}
